### ISSUE TITLE
Training problem

### ISSUE THREAD

**Issue by yrp1234321** [leejiajun/fdscs-pytorch#1]
Thank you for your sharing. 
I've been studying fdscs recently. I trained this network with psmnet, but the network can not converge well. 
Loss will continue to decline, but 3-px error will remain unchanged when it drops to about 70. 
Therefore, could you share your pretraining model or your modifications of training and verification.
Thank you again and look forward to your reply.

**Comment by leejiajun** [comment]
Thank you for following. I've emailed you.
